content/preferences.xul

```diff
@@ -376,13 +376,6 @@
 							but WITHOUT ANY WARRANTY. See the GNU General Public License for more details.
 						</description>
 
-						<description value=""/>
-
-						<description class="ssleuth-pref-text">
-							Jdenticon 1.3.2 zlib license, jdenticon.com (c) Daniel Mester Pirttijärvi
```

Comment:
I doubt removing license disclaimers help…

Reply:
I didn't write that change. It was copied verbatim from the XPI as it appeared on [AMO](https://web.archive.org/web/20180730040927/https://addons.mozilla.org/en-US/firefox/addon/ssleuth/versions/). It appears JDenticon was completely removed in 0.5.3, if you think this was in error I will add it back, but it wasn't in the last published version?